torchrl/envs/transforms/transforms.py

```diff
@@ -539,9 +516,22 @@ def __setattr__(self, key, value):
         if isinstance(value, Transform):
             value.set_parent(self)
         if isinstance(propobj, property):
-            if propobj.fset is None:
+            ancestors = list(__class__.__mro__)[::-1]
```

Comment:
I wonder what this change does? What's the reason for this?

Reply:
The way it was done before did not work if the property was inherited. getattr(self.__class__, key, None) return None if the property is not defined in self.__class__ but in own of its parents.